10-angular/custom_directives/readme.md

```diff
@@ -1,33 +1,27 @@
 # Custom Directives
 
-[Screencast](https://youtu.be/_JKtEbOfX0o)
+## Learning Objectives
 
-## Lenin's Objections
 - DRY a given Angular app by extracting repeating logic and HTML into custom directives
 - Explain the purpose of each of the four directive options, and the four options for the "restrict" directive, E, A, C, M
 - Use a custom directive to render an array of objects
+- Use `link` method to set scope
+- Explain the difference between `@` and `=` in `scope` object
 
-Note that what we cover in class isn't using Firebase; it's picking up where you left off yesterday afternoon.
+[Starter Code](https://github.com/ga-dc/grumblr_angular/releases/tag/3.0.0) | 
+[Solution Code](https://github.com/ga-dc/grumblr_angular/releases/tag/3.0.1)
 
 ```
-git checkout -b templating-and-routing-with-comments origin/templating-and-routing-with-comments 
+// TODO use resource-solution branch
 ```
 
-## This class is about tidying up
-
-That is: breaking things up into separate files.
-
-We approached Angular a bit differently than we did, say, Node. You see how we have all of our controllers in one file? We could totally split those up into separate files, and in Node, we probably would have started off that way. Here, though we're starting with very few files.
-
-Take a look at the Grumbles controller file. It's only 46 lines long and I feel like it's hard to tell which controller is where. It would only take me an extra 2 seconds to figure out, though, which begs the question...
-
-**Why is 2 seconds important? [Relevant visual aid.](http://i.imgur.com/Ssz6pjF.png)**
-
-### All of programming is shortcuts
+```
+$ git checkout -b custom-directives 3.0.0
```

Comment:
this should probably be based off the `resource-solution` branch instead of the 3.0.0 tag.